### PR TITLE
Styling of landing page

### PR DESCRIPTION
**Pull request by zanenkn**
# [PT Story](https://www.pivotaltracker.com/story/show/166374530)

## Changes we propose in this PR:
* Styling of articles on landing page, see screenshots

## What we learned:  
* Adjusting and customising in Semantic UI

## Screenshots
Screen Shot 2019-06-02 at 20 54 04

### CODE REVIEW

**Comment by raptorf1** [comment]
Magnifique!!!!

File: src/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as serviceWorker from './serviceWorker';
 import { BrowserRouter } from 'react-router-dom'
 import axios from 'axios'
 
-axios.defaults.baseURL = 'https://glocal-news.herokuapp.com'
+axios.defaults.baseURL = 'http://localhost:3002/'
```
Comment (leiter007) [inline src/index.js]: Shouldn't this be changed "back" to the deployed site from Heroku, before being merged?

**Review by leiter007 — COMMENTED** [review]
Looks fantastic!! Great styling!

File: src/index.js (hunk 1 above)
Comment (zanenkn) [reply src/index.js]: yes, will do!

File: src/Components/HeaderCategory.jsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
-import { Menu } from 'semantic-ui-react'
+import { Menu, Divider, Container } from 'semantic-ui-react'
 
-const categories = ['News', 'Arts', 'Books', 'Business', 'Food', 'Opinion', 'Politics', 'Real Estate', 'Science', 'Sports', 'Style', 'Tech', 'Travel']
+const categories = [
+  {name: 'News', color: 'red'},
+  {name: 'Business', color: 'blue'},
+  {name: 'Tech', color: 'pink'},
+  {name: 'Sports', color: 'teal'},
+  {name: 'Politics', color: 'grey'},
```
Comment (oliverochman) [inline src/Components/HeaderCategory.jsx:9]: why dont you fetch the categories from the backend?

Comment (zanenkn) [reply src/Components/HeaderCategory.jsx:9]: we are implementing that in another user story (soon to be pr). Can this be merged as is so we avoid unnecessary double work?

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**